languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/hunspell/spelling.txt

```diff
@@ -8096,7 +8096,7 @@ Weckmänner/N
 wegdelegiere/N
 wegdelegierend/A
 wegdelegierst
-wegdelegiert/A
+wegdelegiert
```

Comment:
ne, das würde ich eher nicht machen, glaub ich. dann müsste man das bei allen machen und dann wären ja alle adj-endungen futsch, oder? was meinst du @AgnesKleinhans ?

Reply:
maybe, aber der satz im Diff schlägt sowieso nicht an.

Reply:
Man kann auch eine Logik einbauen, z.B. "wenn das Wort auf 't' endet, dann auch PA-Readings hinzufügen" oder so. Technisch kann ich das machen, bräuchte aber Input von euch, wie genau die Logik aussehen soll.

Reply:
@AgnesKleinhans, @affemitkaraffe und @St-ac-y, lasst uns das morgen früh in unserem Meeting besprechen.

Reply:
@danielnaber: Wir sind zu dem Entschluss gekommen, dass das /A erst mal drinbleiben kann. Ein eventuelles /P (für Partizip 2)  sollten wir nach dem Deployment testen. Könntest du die Funktion für /P morgen nach dem Deployment umsetzen?

Reply:
/P heißt dann nur Partizip oder auch Adjektiv? Man könnte natürlich auch /AP nutzen, wenn man beides will... am besten gebt ihr mir die Tags, die die Wörter mit /p bekommen sollen.